src/compliment/sources.clj

```diff
@@ -23,4 +34,4 @@
   namespace."
   [name & {:as kw-args}]
   {:pre [(every? kw-args [:candidates :doc])]}
-  (swap! sources assoc name (assoc kw-args :enabled true)))
+  (swap! sources assoc name (assoc kw-args :name name :enabled true)))
```

Comment:
include `:name` in the source definition map so that the `:source` selector predicate can also have that info, since it is only provided the keys of the `sources` atom